inc/thorvg.h

```diff
@@ -1177,6 +1201,44 @@ class TVG_EXPORT Picture final : public Paint
      */
     Result load(uint32_t* data, uint32_t w, uint32_t h, bool copy) noexcept;
 
+    /**
+     * @brief Sets the triangle mesh to deform the image.
+     * 
+     * If a mesh is provided, the transform property of the Picture will apply to the triangle mesh, and the
+     * image data will be used as the texture.
+     * 
+     * The vertices of the triangles each contain a point (pt) and a UV coordinate (uv). The uv coordinate is a
+     * point within the image, and must be not fall outside the image. Unlike UVs in OpenGL and some other libraries,
+     * the UV is not between 0 and 1, it is between 0 and the image width/height. i.e.:
+     * 0 <= uv.x <= Picture.width
+     * 0 <= uv.y <= Picture.height
+     * 
+     * Only raster image types are supported at this time (png, jpg). Vector types like svg and tvg do not support
+     * mesh deformation. However, if required you should be able to render a vector image to a raster image and then apply a mesh.
+     * 
+     * @param[in] triangles An array of Polygon objects (triangles) that make up the mesh. Must be at least 1.
+     * @param[in] count The number of Polygon objects (triangles) provided
+     * @param[in] append If false, will replace the mesh completely with the provided mesh. If true, will append the mesh.
```

Comment:
Do it really need the append function?? I'm worrying if it increases the feature-complexity without any practical benefits.

Reply:
@hermet I agree. Will change to be set only.